src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/NuGetExtensions.cs

```diff
@@ -11,10 +11,23 @@ internal static class NuGetExtensions
         {
             if (nugetRef.HasWildcardVersion)
             {
+                if (FloatRange.TryParse(nugetRef.Version, out var range))
```

Comment:
Could you add some unit tests for these? We're looking to fill in some test gaps as much as possible

Reply:
added